src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -864,4 +864,16 @@ private AdsDataProvider() {
     };
   }
 
+  @DataProvider
+  public static Object[][] adsMiddlePrefooter() {
+    return new Object[][]{
+        {
+            "adtest", "", true
+        },
+        {
+            "adtest", "SyntheticTests/Prefooters", false
```

Comment:
what do you think about moving each param to new line? that would increase readability